app/graphql/mutations/create_grading.rb

```diff
@@ -257,7 +257,21 @@ def send_feedback
           timeline_event: submission
         )
 
-      StartupFeedbackModule::EmailService.new(startup_feedback).send
+      grading_details = submission.evaluation_criteria.map.with_index { |criteria, index|
+        grade_label = criteria[:grade_labels].map {|label| label["label"]}
+        grade = submission.timeline_event_grades[index].grade
+        status = grade >= criteria.pass_grade ? "\u{2705}" : "\u{274c}"
```

Comment:
This is something very specific to the view and should be implemented in `feedback_as_email.html.erb` view and not in the mutation.

Reply:
Yes that's right. Now that you will be moving everything to mailer, it will be already handled I suppose.